.git-blame-ignore-revs

```diff
@@ -0,0 +1,3 @@
+# .git-blame-ignore-revs
+# Fix issues from ktlint baseline
+4c6a1f5b46f84349e8405ef5b23dd1381106dc48
```

Comment:
I was double checking if the blame is ignored locally, but I was still seeing this in Android Studio. 

<img width="1728" alt="Screen Shot 2022-09-09 at 11 12 27 AM" src="https://user-images.githubusercontent.com/1190888/189383771-ce9f72ae-aef8-44f6-b228-b77aeff541dc.png">

Reply:
This is for ignoring a commit when assigning blame, not for changing the git log.

Reply:
Ignoring the log panel, you would see the blame in the sidebar for that line of code.

Reply:
Sorry, I figured it out. You have to configure locally to specify the ignore file via `git config blame.ignoreRevsFile .git-blame-ignore-revs`

Reply:
Oh, ok! For that local git must be configured to use the file.

`git config --local blame.ignoreRevsFile .git-blame-ignore-revs`

should do the trick.